homeassistant/components/demo/light.py

```diff
@@ -90,6 +90,14 @@ async def async_setup_platform(
                 supported_color_modes=SUPPORT_DEMO_HS_WHITE,
                 unique_id="light_6",
             ),
+            DemoLight(
```

Comment:
The emulated_hue integration uses the demo integration in the tests and expects a certain state I think. That's why there are some test failures. Same for google_assistant integration.

Reply:
I pushed some fixes for the tests. I agree other integrations shouldn't use demo integration.

Reply:
Wasn't this partially why `kitchen_sink` was made to not clutter `demo` with more entities?

Reply:
Kitchen Sink is about weird and incorrect behavior. Light temperature and brightness are very common entities.